index.js

```diff
@@ -58,7 +59,7 @@ module.exports = {
       'no-caller': 'error',
       'no-case-declarations': 'error',
       'no-else-return': 'error',
-      'no-empty-function': 'error',
+      'no-empty-function': [ 'error', { 'allow': [ 'constructors' ] } ],
```

Comment:
We really just want `'allow': [ 'constructors' ]` for TypeScript, right? We don't want empty constructors in ES code.